app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
```

Comment:
Just as implemented in the comment model, a like should belong to an author😉. kindly fix that👍🏼.
```suggestion
belongs_to :author, class_name: 'User'
```

Reply:
Fixed.

Reply:
Hi @Abenezer-Tilahun  :wave: 
I don't understand you review you just leaved "Fixed" which is unclear for me because I already fixed the mention comment that was provided by @Ibizugbe. :mag:

Reply:
Hello @ahmedeid6842 When I commented Fixed Means that   I wanted to mention that you fixed the suggested changes requested by the previous reviewer.

Reply:
Okay got it :+1: 

Thanks again for your review, But it would be better to mention it in your review as _[Optional changes]_ rather than write "Fixed" in a comment. 

But again many thanks for you time and effort to review my code :pray: , you review is very valuable and awesome. Many thanks @Abenezer-Tilahun :100: :heart_decoration: